### ISSUE TITLE
A11y Fail: Button - loading state - fix roles

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1398]
CENG_1946

https://react-magma.cengage.com/version/4.5.0/api/button/#loading

### Issue:
When a button's state changes to indicate something is loading, this constitutes a status message which requires that it can be "programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus".

### Suggestion:
Please use role="status" on the content that indicates "loading". This may require a text node inside of the `<span>` (instead of an aria-label) in order to function correctly. In addition, it is advised to remove role="img" so that the element continues to be identified as a (disabled) button.

**Comment by chris-cedrone-cengage** [comment]
I see the injected span with an aria-label of "loading", seems good here.
